### PR TITLE
Revert "Merge branch 'main' of https://github.com/adlnet/LDSS-XSS"

### PR DESCRIPTION
**Pull request by vbhayden**
This reverts commit 744abdaff251a365ac9ad362f9fb9efd8c06af0c, reversing changes made to 5efde400ed5ce58b5e67004e21d3ce98173cfa83.

This is a merge commit reversion from the problematic commit last Friday.

### CODE REVIEW

File: app/deconfliction_service/views.py [hunk 1]
```diff
@@ -34,44 +33,32 @@ def run_deconfliction(alias: str, definition: str, context: str, context_descrip
...
-        }
-        return render(request, 'admin/deconfliction_service/deconfliction_admin.html', context)
-    except Exception as e:
-        logger.error(f"Error in deconfliction_admin_view: {e}")
-        messages.error(request, "Error loading deconfliction view")
-        return render(request, 'admin/deconfliction_service/deconfliction_admin.html', {
-            'collisions': [],
-            'duplicates': [],
-            'deviations': [],
-            'non_atomic_definitions': []
```
Comment (bayfaub) [inline app/deconfliction_service/views.py:72]: This should all be kept

File: app/deconfliction_service/views.py [hunk 2]
```diff
@@ -206,85 +193,4 @@ def merge_duplicate_definitions(request, keep_id, remove_id):
...
-        term_node.deprecated = True
-        term_node.save()
-
-        messages.success(request, "Successfully deprecated definition.")
-        return redirect('admin:admin_deconfliction_view')
-
-    except Exception as e:
-        logger.error(f"Error deprecating definition: {e}")
-        messages.error(request, f"Error deprecating definition: {str(e)}")
-        return redirect('admin:admin_deconfliction_view')
```
Comment (bayfaub) [inline app/deconfliction_service/views.py:290]: This should all be kept.

File: app/openlxp_xss_project/urls.py [hunk 3]
```diff
@@ -18,10 +18,8 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path
-from django.shortcuts import redirect
 
 urlpatterns = [
-    path('', lambda request: redirect('admin/', permanent=True)),
```
Comment (bayfaub) [inline app/openlxp_xss_project/urls.py:24]: This should be kept.

**Review by bayfaub — CHANGES_REQUESTED** [review]

**Comment by vbhayden** [comment]
Yeah I'm not sure why so many of the more recent changes were rolled back with the reversion of this merge.  The commit prior to it had everything there...

**Review by bayfaub — APPROVED** [review]

**Pull request merged**